project/context.go

```diff
@@ -22,6 +22,7 @@ type Context struct {
 	ComposeFiles        []string
 	ComposeBytes        [][]byte
 	ProjectName         string
+	EnvName             string
```

Comment:
Minor nit, can you change `Env` to `Environment` here and other places where it's shortened?